agent.go

```diff
@@ -36,12 +37,12 @@ func (b ByTime) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
 func (b ByTime) Less(i, j int) bool { return b[i].LamportTime < b[j].LamportTime }
 
 type agent struct {
-	networkDB         *networkdb.NetworkDB
-	bindAddr          string
-	advertiseAddr     string
-	dataPathAddr      string
-	epTblCancel       func()
-	driverCancelFuncs map[string][]func()
+	networkDB          *networkdb.NetworkDB
+	bindAddr           string
+	advertiseAddr      string
+	dataPathAddr       string
+	epTblCancelFunList []func()
```

Comment:
It is not only for endpoint tables, it is also for node event tables.
Please rename the variable to `coreCancelFuncs`. We already have a `driverCancelFuncs`, so that it is clear this new one contains the cancel functions for the tables followed by libnetwork core